t/nextjs_react_template/contents/src/app/navbar.tsx

```diff
@@ -0,0 +1,21 @@
+import { AppBar, Toolbar, Typography } from "@mui/material";
+import Auth from "./auth";
+import { APP_TITLE } from "./constants";
+
+export default function NavBar() {
+  return (
+    <AppBar position="static" sx={{ bgcolor: "#202124" }}>
```

Comment:
Curious why we want this color, does the default AppBar not work? For hard coded color, usually it's also put into constants. But we can use the default to avoid the overhead

Reply:
Only because the default Auth0 picture is also blue, so there is very little contrast between AppBar background and the avatar. Don't feel too strongly about it, so I removed the override.